docs/lazy.md

```diff
@@ -148,6 +148,24 @@ $di->setters['Example']['addBars'] = $di->lazyArray([
 ]);
 ```
 
+If you need to modify the LazyArray after you've assigned it we provide two methods (`append($value, $key = null)` and `getArrayCopy()` to help.
```

Comment:
Key, Value signature flipped is something I don't like. But I could not suggest an alternative here. So I am sad :( .